src/test/java/com/ericsson/ei/queryservice/test/QueryServiceRESTAPITest.java

```diff
@@ -98,12 +98,13 @@ public void getQueryAggregatedObjectTest() throws Exception {
                 expectedOutputWithSquareBrackets.length() - 1));
         JsonNode expectedOutput = mapper.readTree(expectedOutputString);
 
-        Mockito.when(aggregatedObjectController.getQueryAggregatedObject(Mockito.anyString(), Mockito.any(HttpServletRequest.class)))
+        Mockito.when(aggregatedObjectController.getAggregatedObjectById(Mockito.anyString(),
+                Mockito.any(HttpServletRequest.class)))
                 .thenReturn(new ResponseEntity(response.get(0), HttpStatus.OK));
 
-        RequestBuilder requestBuilder = MockMvcRequestBuilders.get("/queryAggregatedObject")
-                .accept(MediaType.APPLICATION_JSON).param("ID", "6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43")
-                .contentType(MediaType.APPLICATION_JSON);
+        RequestBuilder requestBuilder = MockMvcRequestBuilders
+                .get("/aggregated-objects/" + "6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43")
+                .accept(MediaType.APPLICATION_JSON).contentType(MediaType.APPLICATION_JSON);
```

Comment:
Below is a test to query missed notification, is this not done in other tests and should not be here? Please check this up.

Reply:
I havent seen it anywhere else. I agree this test shouldnt be here. It is here (I suppose) because of the word "query".  So I am not sure should it be moved to notifications related test or .... any suggestion?